deployments/kovan/migrations/1644388553_deploy_kovan.ts

```diff
@@ -51,19 +52,25 @@ migration('1644388553_deploy_kovan', {
       cloneNetwork
     );
 
-    await wait(await usdcProxy.changeAdmin(usdcProxyAdmin.address));
+    debug(`Changing admin of USDC proxy to ${usdcProxyAdmin.address}`);
```

Comment:
I've added more debug logs to make it even clearer which txns are being run and retried.